soroban-sdk-auth/src/lib.rs

```diff
@@ -0,0 +1,112 @@
+#![no_std]
+
+use soroban_sdk::{serde::Serialize, Account, BigInt, BytesN, Env, EnvVal, Symbol, TryIntoVal};
+
+pub mod public_types;
+use crate::public_types::{
+    Identifier, KeyedAccountAuthorization, KeyedAuthorization, KeyedEd25519Signature, Message,
+    MessageV0,
+};
+
+pub trait NonceAuth {
+    fn read_nonce(e: &Env, id: Identifier) -> BigInt;
+    fn read_and_increment_nonce(&self, e: &Env, id: Identifier) -> BigInt;
+    fn get_keyed_auth(&self) -> &KeyedAuthorization;
+}
+
+pub fn check_ed25519_auth(
+    e: &Env,
```

Comment:
In most places in the SDK we use `env` for the `Env`, and `e` for errors in small scopes. Could you rename this toe `env` for consistency?

Reply:
Done